README.md

```diff
@@ -201,6 +201,19 @@ Please review the [attributes/default.rb](https://github.com/DataDog/chef-datado
 
 For general information on the Datadog Agent 6, please refer to the [datadog-agent](https://github.com/DataDog/datadog-agent/) repo.
 
+#### Windows Agent v6 installation
+
+Starting with version `>= 6.11`, the Agent v6 must be installed with a chef-datadog
+cookbook version `>= 2.18.0`.
+
+For Agent v6 versions `< 6.11`, previous versions of the datadog cookbook
+could still be used, even if we recommend in most of the case to use the
```

Comment:
this paragraph can be removed I think: overall even if it may work in some cases we know the standard use case will be broken, so no need to go into details IMO :)